CONVERTER-SERVICE/__test__/imageConverter.model.test.js

```diff
@@ -0,0 +1,36 @@
+const ImageConverter = require('../src/models/imageConverter.model');
+const fs = require('fs');
+
+describe('Image Converter tests', () => {
+  const saveImagesPath = `${__dirname}/files/imagesFilesFolder`;
+  const size = {
+    width: Number(100) || null,
```

Comment:
I htink you should not converter and validate to a null. It has to be tested in the tests